vscode-dotnet-runtime-library/src/test/unit/DebianDistroTests.test.ts

```diff
@@ -0,0 +1,185 @@
+/* --------------------------------------------------------------------------------------------
+ *  Licensed to the .NET Foundation under one or more agreements.
+*  The .NET Foundation licenses this file to you under the MIT license.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as chai from 'chai';
+import * as os from 'os';
+import { MockCommandExecutor, MockEventStream } from '../mocks/MockObjects';
+import { DistroVersionPair, DotnetDistroSupportStatus, LinuxVersionResolver } from '../../Acquisition/LinuxVersionResolver';
+import { getMockAcquisitionContext, getMockUtilityContext } from './TestUtility';
+import { DotnetInstallMode } from '../../Acquisition/DotnetInstallMode';
+import { DebianDistroSDKProvider } from '../../Acquisition/DebianDistroSDKProvider';
+const assert = chai.assert;
+const standardTimeoutTime = 100000;
+
+const mockVersion = '7.0.103';
+const acquisitionContext = getMockAcquisitionContext('sdk', mockVersion);
+const mockExecutor = new MockCommandExecutor(acquisitionContext, getMockUtilityContext());
+const pair: DistroVersionPair = { distro: 'Debian', version: '12' };
+const provider: DebianDistroSDKProvider = new DebianDistroSDKProvider(pair, acquisitionContext, getMockUtilityContext(), mockExecutor);
+const shouldRun = os.platform() === 'linux';
+const installType: DotnetInstallMode = 'sdk';
+const noDotnetString = `
+dotnet: command not found
```

Comment:
@nagilson can i have suggestion for noDotnetString value? what is the usage of it?

Reply:
This is a sample value of what the distro would return if you tried calling 'dotnet --info' or any dotnet command when it's not installed. It's used for testing to make sure the parsing logic doesn't fail on it.

Reply:
so we don't need anychange, this message is result of calling dotnet when it is not installed